docs/Formula-Android.md

```diff
@@ -0,0 +1,380 @@
+## Formula Android
+The Android module provides a declarative API to connect reactive state management to Android Fragments. 
+This module has been designed for gradual adoption. You can use as much or as little of it as you like.
+
+Some of the goals for this module are:
+
+    - Use single RxJava stream to drive the UI.
+    - Separate state management from Android UI lifecycle.
+    - Ability to group multiple fragments into a flow and share state between them.
+    - Type-safe and scoped fragment event handling. (Avoid casting activity to a listener)
+
+## Using Fragments
+This module provides an API to connect state management and view rendering logic to Android 
+fragments. For this example, we will connect `CounterRenderView` and `CounterFormula` from the 
+main getting started [guide](index.md). 
+
+### Define a fragment key
+Fragment key is used an a unique identifier for a `FormulaFragment`.
```

Comment:
How unique does it have to be? What if I use the same fragment key twice?

Reply:
I'm updating this to the following

> Fragment key is used to instantiate `FormulaFragment` and to identify which `FeatureFactory` to 
use. You can also use it to add arguments that the fragment instance needs.